WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -663,6 +670,11 @@ private extension ProductFormViewController {
         }
     }
 
+    func createPublishBarButtonItem() -> UIBarButtonItem {
+        let updateTitle = NSLocalizedString("Publish", comment: "Action for creating a new Product remotely")
+        return UIBarButtonItem(title: updateTitle, style: .done, target: self, action: #selector(updateProduct))
+    }
```

Comment:
```suggestion
    func createPublishBarButtonItem() -> UIBarButtonItem {
        let publishTitle = NSLocalizedString("Publish", comment: "Action for creating a new Product remotely")
        return UIBarButtonItem(title: publishTitle, style: .done, target: self, action: #selector(updateProduct))
    }
```

Reply:
oops! updated in https://github.com/woocommerce/woocommerce-ios/pull/2846/commits/19396ff0470831b2f357f600d2619d1f8373dfe5 😄